.github/workflows/actions.yml

```diff
@@ -10,14 +10,12 @@ jobs:
       fail-fast: false
       matrix:
         ruby-version:
-          - "2.7"
-          - "3.0"
-          - "3.1"
           - "3.2"
```

Comment:
should keep still valid versions 3.0/3.1 and possibly add 3.3 too

Reply:
Thank you!